lib/components/atoms/Image/Image.story.js

```diff
@@ -26,7 +26,18 @@ storiesOf('Atoms', module).addWithChapters('Image', {
       sections: [
         {
           title: 'Default',
-          sectionFn: () => <Image {...defaultImage} />,
+          sectionFn: () => {
+            return <Image {...defaultImage} />;
+          },
+        },
+      ],
+    },
+    {
+      title: 'Lazy loaded Image Variations',
```

Comment:
If the sample is already present in viewport the Demo is kind of incomplete. 

Do this: 

```
sections: [
        {
          title: 'Default lazy loaded',
          sectionFn: () => <p style="height:500px; color:blue;">Scroll Down to lazy load image</p>
        },
{
          title: 'This image will lazy load, check in network tab of chrome dev tools',
          info: 'Note: this only works in Chrome so far, check for updates here https://caniuse.com/#search=loading',
          sectionFn: () => <Image {...lazyLoadedImage} />,
        },
      ],
```